string move_group_name
string pose_name
string planner_name
---
